StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodParams.swift

```diff
@@ -116,6 +116,9 @@ public class STPPaymentMethodParams: NSObject, STPFormEncodable {
     /// Set of key-value pairs that you can attach to the PaymentMethod. This can be useful for storing additional information about the PaymentMethod in a structured format.
     @objc public var metadata: [String: String]?
 
+    /// Contains metadata with identifiers for the session and information about the integration
+    @objc @_spi(STP) public var clientAttributionMetadata: STPClientAttributionMetadata = STPClientAttributionMetadata()
```

Comment:
This might not need to be `@objc`. Same goes for `STPClientAttributionMetadata`. If true, feel free to address to a future PR to avoid having to block more on this PR.